SampleCode-V5/android-sdk-v5-uxsdk/src/main/java/dji/v5/ux/pachWidget/PachWidgetModel.java

```diff
@@ -0,0 +1,54 @@
+package dji.v5.ux.pachWidget;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+public class PachWidgetModel extends ViewModel{
+    private static PachWidgetModel instance = null;
+    private IPachWidgetModel pachKeyManager;
+    private MutableLiveData<String> _msgdata;
+    protected LiveData<String> msgdata;
+    private MutableLiveData<Boolean> _connectiondata;
+    protected LiveData<Boolean> connectiondata;
+    private PachWidgetModel() {
```

Comment:
If this was built from a previous widget model, please reference that here.

Reply:
Descriptions of why these variables exist have been added.